src/server.js

```diff
@@ -39,7 +51,13 @@ class Server extends EventEmitter {
 	}
 
 	parse(message, rinfo) {
-		const packet = NTPPacket.parse(message);
+		let packet;
+		try {
+			packet = NTPPacket.parse(message);
+		} catch (error) {
+			this._onInvalidPacket(message, error)
+			return
+		}
```

Comment:
You can improve this by getting all the process that the parse function does and putting it on the try block, leaving only the this._onInvalidPacket(message, error) in the catch block.
```suggestion
		try {
			const packet = NTPPacket.parse(message);
			const rxTimestamp = Math.floor(Date.now() / 1000);

  		        packet.originateTimestamp = Math.floor(packet.txTimestamp);
		        packet.referenceTimestamp = rxTimestamp - 5;
		        packet.rxTimestamp = rxTimestamp;
		        this.emit('request', packet, this.send.bind(this, rinfo));
		} catch (error) {
			this._onInvalidPacket(message, error)
		}
```